api/src/paths/project/{projectId}/view.ts

```diff
@@ -85,7 +85,7 @@ GET.apiDoc = {
  *
  * @returns {RequestHandler}
  */
-function getProjectForView(): RequestHandler {
+export function getProjectForView(): RequestHandler {
```

Comment:
why this?

Reply:
ah, I was trying to reuse this function, but ended up not using it.  Removing it.  Nice catch!